chunky/src/java/se/llbit/chunky/model/PitcherCropTopModel.java

```diff
@@ -0,0 +1,168 @@
+package se.llbit.chunky.model;
+
+import se.llbit.chunky.resources.Texture;
+import se.llbit.math.Quad;
+import se.llbit.math.Vector3;
+import se.llbit.math.Vector4;
+
+public class PitcherCropTopModel extends QuadModel {
+  private static final Texture pitcher_top = Texture.pitcherCropTop;
+  private static final Texture pitcher_side = Texture.pitcherCropSide;
+  private static final Texture pitcher_bottom = Texture.pitcherCropBottom;
+  private static final Texture stage_2 = Texture.pitcherCropBottomStage2;
+
+  private static final Texture[][] textures = new Texture[][]{
+    null, null,
+    new Texture[]{
+      stage_2, stage_2, stage_2, stage_2, pitcher_top, pitcher_bottom, pitcher_side, pitcher_side, pitcher_side, pitcher_side
+    },
+    new Texture[]{
+      Texture.pitcherCropTopStage3, Texture.pitcherCropTopStage3, Texture.pitcherCropTopStage3, Texture.pitcherCropTopStage3
```

Comment:
Any reason why stage_2 is an instance variable above, but not these stage textures?

Reply:
First, this model didn't have a model for age=2. But then I saw that the bottom model of age=2 is higher than one block, so I added the same model to the top model (translated by 1 block to the bottom). Same trick as for pitcher plant.

I'll make these instance variables, too.